### PR TITLE
Allow gemm profile by pass args from commandline

### PR DESCRIPTION
**Pull request by cloudhan**
**Description**: Describe your changes.

This allow us quickly launch a microbench session by, for example:
```bash
python gemm_test.py T N float16 256 256 65536 
```
So that we can quickly see which one is the fastest.

### CODE REVIEW

**Comment by cloudhan** [comment]
This allow us quickly launch a microbench session by, for example:
```bash
python gemm_test.py T N float15 256 256 65536 
```
So that we can quickly see which one is the fastest.

**Comment by cloudhan** [comment]
@PeixuanZuo I think we shall make all this profile script to support profile from commandline.

File: onnxruntime/python/tools/kernel_explorer/kernels/kernel_explorer.pyi [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class DeviceArray:
```
Comment (zhangyaobit) [inline onnxruntime/python/tools/kernel_explorer/kernels/kernel_explorer.pyi:1]: Do we need this file?

**Comment by zhangyaobit** [comment]
> @PeixuanZuo I think we shall make all these profile scripts to support profile from commandline.

Sounds great. Could be in a separate PR though.

File: onnxruntime/python/tools/kernel_explorer/kernels/kernel_explorer.pyi (hunk 1 above)
Comment (cloudhan) [reply onnxruntime/python/tools/kernel_explorer/kernels/kernel_explorer.pyi:1]: Yeah, otherwise pyright lint will dump garbarge in the Files changed tab

**Review by zhangyaobit — APPROVED** [review]

**Review by PeixuanZuo — APPROVED** [review]

**Pull request merged**